src/lib/nodes/btc/BtcNode.ts

```diff
@@ -29,4 +41,18 @@ export class BtcNode extends Node {
       ping: Date.now() - time
     }
   }
+
+  private async fetchNodeVersion(): Promise<void> {
+    try {
+      const { data } = await this.client.post<FetchBtcNodeInfoResult>('/bitcoind', {"jsonrpc":"1.0","id":"adm","method":"getnetworkinfo","params":[]})
+      const v =  data?.result?.version
+      if (v) {
+        this.version = 'v' + formatBtcVersion(v)
+      } else {
+        console.error(data.error)
+      }
+    } catch (e) {
```

Comment:
Remove `try {} catch {}` block. The request can fail, thats ok, it will leave an empty version. If catching error will be needed, this can be done outside the function.